src/materialized/src/bin/materialized/main.rs

```diff
@@ -105,9 +105,10 @@ struct Args {
     /// Set to "off" to disable logical compaction.
     #[structopt(long, env = "MZ_LOGICAL_COMPACTION_WINDOW", parse(try_from_str = parse_optional_duration), value_name = "DURATION", default_value = "1ms")]
     logical_compaction_window: OptionalDuration,
-    /// [DEPRECATED] Frequency with which to advance timestamps.
-    #[structopt(long, env = "MZ_TIMESTAMP_FREQUENCY", hidden = true, parse(try_from_str = parse_duration::parse), value_name = "DURATION", default_value = "10ms")]
+    /// Timestamp frequency to apply to sources
```

Comment:
The docs have the following for the per-source `timestamp_frequency_ms`
```
Default: 1000. Sets the timestamping frequency in ms. Reflects how frequently timestamps advance in the system. 
```

So maybe the help text for this should be `Default frequency with which to advance timestamps`? 

And I guess we should eventually update the docs (no need to do that in this pr)

Reply:
👍🏽 let's do in this PR so we don't forget though.

Reply:
(Also, I think we should say "the default frequency at which sources advance their timestamps" or something like that.)